src/http-spec.ts

```diff
@@ -123,12 +121,15 @@ export interface IHttpCookieParam extends IHttpParam {
  */
 
 export interface IHttpContent {
-  mediaType: string;
   schema?: ISchema;
   examples: Array<INodeExample | INodeExternalExample>;
   encodings: IHttpEncoding[];
```

Comment:
I think that the Media Content interface does not need this property. Maybe we could use `Omit`?

Reply:
I think it does :). Otherwise no way to associate the content with a media type in places like request body etc?

Reply:
Oh you're right. I was convinced I didn't write this at the end 🤔